### PR TITLE
Add react (with styles)

### PR DESCRIPTION
**Pull request by bloodyowl**

### CODE REVIEW

File: packages/benchmarks/react-styles/package.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "license": "MIT",
+  "dependencies": {
+    "benchmarks-utils": "1.0.0"
+  },
+  "devDependencies": {
+    "dev-tasks": "1.0.0"
+  },
+  "benchmarks": {
+    "name": "react (with styles)",
+    "link": "https://github.com/facebook/react"
```
Comment (A-gambit) [inline packages/benchmarks/react-styles/package.json]: Please add:
```
 "useInlineStyles": true
```

**Comment by A-gambit** [comment]
Please rename `react-styles` to `react-inline-styles`

**Comment by A-gambit** [comment]
Hey @bloodyowl, please make appropriate changes!

**Comment by bloodyowl** [comment]
@A-gambit sorry about the late fix, done now

**Comment by A-gambit** [comment]
@bloodyowl I am sorry, but the build went down :(

**Comment by bloodyowl** [comment]
@A-gambit should be ok now 😄

**Pull request merged**